src/entities/Period.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose';
+
+export interface PeriodInput {
+  name: string;
+  status: string; // OPEN, QUANTIFY, CLOSED
+  endDate: Date;
+  quantifiers: [string];
+}
+
+export interface PeriodDocument extends PeriodInput, mongoose.Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const periodSchema = new mongoose.Schema(
+  {
+    id: { type: String, required: true, unique: true },
```

Comment:
Doesn't Mongoose create a `_id` for you automatically? If so we don't need this id.